.github/workflows/build-and-test.yml

```diff
@@ -9,5 +9,5 @@ on:
 
 jobs:
   build-and-test:
-    uses: GTNewHorizons/GTNH-Actions-Workflows/.github/workflows/build-and-test.yml@master
+    uses: GTNewHorizons/GTNH-Actions-Workflows/.github/workflows/build-and-test.yml@rfg
```

Comment:
The actions changes are temporary until we move mods over to the new buildscript, they're only needed for Jabel which is opt-in (just on by default for new projects including this template)